src/metakb/cli.py

```diff
@@ -7,401 +7,366 @@
 from os import environ
 from pathlib import Path
 from timeit import default_timer as timer
-from typing import Optional, Set, Union
+from typing import Optional
 from zipfile import ZipFile
 
 import asyncclick as click
 import boto3
 from boto3.exceptions import ResourceLoadException
 from botocore.config import Config
-from disease.cli import CLI as DiseaseCLI  # noqa: N811
-from disease.database import Database as DiseaseDatabase
-from disease.schemas import SourceName as DiseaseSources
-from gene.cli import CLI as GeneCLI  # noqa: N811
-from gene.database import Database as GeneDatabase
-from gene.schemas import SourceName as GeneSources
-from therapy.cli import CLI as TherapyCLI  # noqa: N811
-from therapy.database import Database as TherapyDatabase
-from therapy.schemas import SourceName as TherapySources
+from disease.cli import update_db as update_normalizer_disease_db
+from gene.cli import update_normalizer_db as update_normalizer_gene_db
+from therapy.cli import update_normalizer_db as update_normalizer_therapy_db
 
 from metakb import APP_ROOT
 from metakb.database import Graph
-from metakb.harvesters import CivicHarvester, Harvester, MoaHarvester
+from metakb.harvesters.civic import CivicHarvester
+from metakb.harvesters.moa import MoaHarvester
 from metakb.schemas.app import SourceName
-from metakb.transform import CivicTransform, MoaTransform, Transform
+from metakb.transform import CivicTransform, MoaTransform
 
-logger = logging.getLogger("metakb.cli")
-logger.setLevel(logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+S3_CDM_PATTERN = re.compile(r"cdm/20[23]\d[01]\d[0123]\d/(.*)_cdm_(.*).json.zip")
```

Comment:
this can probably be moved within the scope of `_retrieve_s3_cdms` since that's a function that will presumably be run no more than once per interpreter session and there don't appear to be any other references to it